components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -1545,6 +1545,20 @@ def getInterpolateSetting(self):
         return (self.getConfigService().getConfigValue(
                 "omero.client.viewer.interpolate_pixels") or 'true')
 
+    def getDownloadAsJpgMaxSizeSetting(self):
+        """
+        Returns default max size of images that can be downloaded as
+        jpg, png or tiff, expressed as number of pixels.
+        Default is 144000000 (12k * 12k image)
+
+        :return:    Integer
+        """
+        size = self.getConfigService().getConfigValue(
```

Comment:
The same here, it is breaking change for 5.1 series. It should go in try catch block to guarantee compatibility with < 5.1.4 server.